pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1926,6 +1926,24 @@ public boolean hasOfflineTable(String tableName) {
     return getAllTables().contains(actualTableName);
   }
 
+  /**
+   * Gets the ideal state of the table
+   * @param tableNameWithType
```

Comment:
[Minor] If we are not going to add doc for param and return, remove these 2 lines.